package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rebilly-js-sdk",
-  "version": "15.3.5",
+  "version": "15.3.6",
```

Comment:
The version needs to be updated to `15.3.7`

Reply:
thank you, updated.